server/datastore/mysql/migrations/tables/20230906134103_AddRetriesColumnHostMdmAppleProfilesTable.go

```diff
@@ -0,0 +1,25 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230906134103, Down_20230906134103)
+}
+
+func Up_20230906134103(tx *sql.Tx) error {
+	stmt := `
+ALTER TABLE host_mdm_apple_profiles
+	ADD COLUMN retries int(2) UNSIGNED NOT NULL DEFAULT 0`
```

Comment:
Just a heads-up, I think it's ok to use an `int` for this but regarding the parens `(2)`, it's often misinterpreted - this is just a hint of how to display the field, it's not constraining the size of the integer (https://dev.mysql.com/doc/refman/8.0/en/numeric-type-attributes.html):

> The display width does not constrain the range of values that can be stored in the column. Nor does it prevent values wider than the column display width from being displayed correctly. For example, a column specified as [SMALLINT(3)](https://dev.mysql.com/doc/refman/8.0/en/integer-types.html) has the usual [SMALLINT](https://dev.mysql.com/doc/refman/8.0/en/integer-types.html) range of -32768 to 32767, and values outside the range permitted by three digits are displayed in full using more than three digits.

If you intended to use a smaller-size integer, you'd need to use `TINYINT` (https://dev.mysql.com/doc/refman/8.0/en/integer-types.html).